test_files/preprocessor.h

```diff
@@ -0,0 +1,4 @@
+
+#define FOO 0
+#define FOOBAR 1
+#define FOOBARBAZ 2
```

Comment:
Actually it should work for more things than integers. I've just added tests for strings.